tools/install.py

```diff
@@ -182,8 +187,9 @@ def headers(action):
       'deps/zlib/zlib.h',
     ], 'include/node/')
 
-def run(args):
+def run(opts, args):
   global node_prefix, install_path, target_defaults, variables
+  components = set(['node', 'npm', 'headers'])
```

Comment:
@rvagg Ah, you are correct. Currently also, when we install node and headers, we install npm only if 'node_install_npm' is set. Good catch. I'll fix it.